marimbabot_speech/src/speech2text.py

```diff
@@ -1,4 +1,6 @@
 import os
+from nvidia import cudnn
+os.environ['LD_LIBRARY_PATH'] = f'{os.path.dirname(cudnn.__file__)}/lib:$LD_LIBRARY_PATH'
```

Comment:
Can you add a comment why this change is needed?

Reply:
since I haven't specified the version of TensorFlow, it seems to install the newest version, so according to [here](https://www.tensorflow.org/install/pip), the environment needs to be configured.

Reply:
Okay :+1: please add this as a comment next to the loc

Reply:
You mean to add the comment next to this code, right?